src/dev-app/theme.scss

```diff
@@ -60,3 +61,7 @@ $dark-theme: mat-dark-theme($dark-primary, $dark-accent, $dark-warn);
   @include mat-strong-focus-indicators-theme($dark-theme);
   @include mat-mdc-strong-focus-indicators-theme($dark-theme);
 }
+
+.demo-dialog-legacy-padding {
+  @include mat-mdc-dialog-legacy-padding();
```

Comment:
What's the context behind putting this in the theme for the dev app?

Reply:
Moved it into the demo. Initially put it there due to view encapsulation